app/src/main/res/layout/lesson_thumbnail_image_view_test_fragment.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
we can skip using layout, if no data binding work required

Reply:
Done